Dfe.Academies.External.Web/Models/SchoolApplyingToConvert.cs

```diff
@@ -8,6 +8,12 @@ public SchoolApplyingToConvert(string schoolName, int urn, int applicationId, st
 		UKPRN = ukprn;
 		URN = urn;
         ApplicationId = applicationId;
+        LandAndBuildings = new(ApplicationId:applicationId,
+								URN:urn, 
```

Comment:
Does the LandAndBuildings object need the URN?

Reply:
in theory no. I was thinking the same as I was wiring that up to generate an empty L & B object